<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadSignalDialog_C</class>
 <widget class="QDialog" name="LoadSignalDialog_C">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="_main_grid_layout">
    <item row="2" column="0">
     <spacer name="_vertical_spacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="_control_layout_horizontal">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="_btn_load_from_disk">
        <property name="text">
         <string>Load from Disk</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_load_serial_port">
        <property name="text">
         <string>New SerialPort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_remove_signal">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_ok">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_btn_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="_label_signals">
      <property name="text">
       <string>Signals</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="_tree_view_layout_horizontal">
      <item>
       <widget class="QTreeView" name="_tree_view_loaded_signals"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
